app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -14,6 +14,19 @@ class VerifyCsrfToken extends BaseVerifier
     protected $except = [
         '/User/Settings/Ajax',
         '/Microblog/Comment/Show/*',
-        '/Forum/Comment/*'
+        '/Forum/Comment/*',
     ];
+
+    public function handle($request, \Closure $next)
```

Comment:
Ten kod nie bedzie potrzebny. Laravel potrafi wykonac rozne middleware, w zaleznosci od kontekstu (czy dany kontroler jest wykonywany poprzez API czy WEB). Mozesz zajrzec do pliku `app\Http\Kernel.php` (linia 23).

Trzeba bedzie za to dokonac jednej malej modyfikacji. W pliku `RouteServiceProvider.php` . W metodzie `mapApiRoutes()` trzeba dodac:

```php
$this->router->group([
            'namespace' => $this->namespace, 'middleware' => 'api',
        ], function () {
            require base_path('routes/api.php');
        });
```

Linie `require base_path('routes/api.php');` trzeba usunac z `mapWebRoutes()`.